models/classes/resources/ListResourceLookup.php

```diff
@@ -74,7 +74,13 @@ private function searchByString($searchString, $rootClass, $offset, $limit)
         /** @var ResultSet $result */
         $result = $searchService->query($searchString, $rootClass, $offset, $limit);
         $count = $result->getTotalCount();
-        return $this->format($result, $count, $offset, $limit);
+
+        $ids = [];
+        foreach ($result as $item) {
```

Comment:
This would also do the trick, but it's up to you :)
```suggestion
        $ids = array_column($result, 'id');
```